lib/prefab/client.rb

```diff
@@ -99,8 +99,8 @@ def on_update(&block)
       resolver.on_update(&block)
     end
 
-    def log_internal(level, msg, path = nil)
-      log.log_internal msg, path, nil, level
+    def log_internal(level, msg, path = nil, **tags)
```

Comment:
what do you think about calling this `log_context` instead of `tags`. Tags feels like `["tag1", "tag2"]`

Reply:
That works for me, and makes more sense! I'll make the change this evening

Reply:
Done, plus added one more test to ensure that this all works if you pass the logger message as a block!